src/insertWord.js

```diff
@@ -4,10 +4,6 @@ import WordService from "./service/WordService";
 const insertUi = new InsertUI(document);
 const wordService = new WordService();
 
-insertUi.init();
-
-insertUi.clearBtn.addEventListener("click", () => insertUi.clear());
-
 insertUi.addWordBtn.addEventListener("click", () => {
   const word = insertUi.createWord();
```

Comment:
Need check whether the returned `word` is `null` or not.
If it is `null` you should stay on the same page, or even better, showing some alert messages.